components/brave_vpn/browser/api/BUILD.gn

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2022 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//brave/components/brave_vpn/buildflags/buildflags.gni")
+
+assert(enable_brave_vpn)
+
+source_set("api") {
+  sources = [
+    "brave_vpn_api_helper.cc",
+    "brave_vpn_api_helper.h",
+    "brave_vpn_api_request.cc",
+    "brave_vpn_api_request.h",
+    "vpn_response_parser.cc",
+    "vpn_response_parser.h",
+  ]
+
+  deps = [
+    "//base",
+    "//brave/components/api_request_helper:api_request_helper",
+    "//brave/components/brave_vpn/common",
+    "//brave/components/brave_vpn/mojom",
+    "//brave/components/skus/browser",
+    "//components/prefs",
+    "//net/traffic_annotation",
+    "//services/network/public/cpp",
+    "//url",
+  ]
+}
+
+source_set("unit_tests") {
+  testonly = true
+
+  sources = []
+
+  if (is_win || is_mac || is_android) {
```

Comment:
this guard is not appropriate because brave_vpn_api_helper and vpn_response_parser do not have os guards

Reply:
removed